core/src/test/scala/unit/kafka/admin/FeatureCommandTest.scala

```diff
@@ -107,4 +116,17 @@ class FeatureCommandTest extends BaseRequestTest {
       assertTrue(downgradeDescribeOutput.contains(expectedOutput))
     }
   }
+
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
+  @ValueSource(strings = Array("kraft"))
+  def testUpdateFeatureByReleaseVersion(quorum: String): Unit = {
+    val initialDescribeOutput = TestUtils.grabConsoleOutput(FeatureCommand.mainNoExit(Array("--bootstrap-server", bootstrapServers(), "describe")))
+    assertTrue(initialDescribeOutput.contains("Feature: metadata.version\tSupportedMinVersion: 1\tSupportedMaxVersion: 7\tFinalizedVersionLevel: 1"))
+
+    FeatureCommand.mainNoExit(Array("--bootstrap-server", bootstrapServers(), "upgrade", "--release", IBP_3_3_IV3.version()))
```

Comment:
Can we add a negative test case where `--release` and `--feature` are given?

Reply:
I moved the original `FeatureCommandTest` to `FeatureCommandIntegrationTest` and add a new `FeatureCommandTest` to do these unit tests.